src/store/counter/userSlice.js

```diff
@@ -84,11 +77,12 @@ export const signIn = createAsyncThunk(
         );
       })
       .catch((error) => {
-        data.setErrors(`Failed to sign in ${error.message}`);
-        const { code } = error;
         const { message } = error;
-        const { email } = error;
-        console.log(code, message, email);
+        message === "Firebase: Error (auth/user-not-found)."
+          ? data.setErrors(`user not found`)
+          : message === "Firebase: Error (auth/wrong-password)."
+          ? data.setErrors(`Wrong Password`)
+          : data.setErrors(``);
```

Comment:
This supposed to show a message but doesn't work for me when I login with a wrong password or an account that doesn't work.

Reply:
done